client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -814,6 +814,12 @@ const Editor = (props: EditorProps): JSX.Element => {
     Prism.hooks.add('complete', makePrismCollapsible);
     Prism.highlightAllUnder(desc);
 
+    const obs = new ResizeObserver(() => {
+      updateDescriptionZone();
+    });
+
```

Comment:
```suggestion
    // Since the description can be resized without React knowing about it, the
    // zone needs updating in response.
    const obs = new ResizeObserver(() => updateDescriptionZone());
```
Some clarification might help.